.ci/prepare_env_utils.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2017 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+function installDependencies() {
+# Getting core repos ready
+	yum install --assumeyes epel-release
+	yum update --assumeyes
+	yum install --assumeyes python-pip
+	
+	# Test and show version
+	pip -V
+	
+	# Getting dependencies ready
+	yum install --assumeyes \
+	            docker \
+	            git \
+	            patch \
+	            pcp \
+	            bzip2 \
+	            golang \
+	            make \
+	            jq \
+	            java-1.8.0-openjdk \
+	            java-1.8.0-openjdk-devel \
+	            centos-release-scl
+	
+	yum install --assumeyes \
+	            rh-maven33 \
+	            rh-nodejs4
```

Comment:
Does it make sense to use `rh-nodejs8` here? The docs are a bit in disagreement, but I had to use Node 8 for the pr-check plugin.

https://github.com/eclipse/che/tree/master/dashboard#quick-start

Reply:
I'm not sure what effects does switching to nodejs8 have, however if it's working with the prcheck, I don't see any issues there. we can use rh-nodejs8 instead.

Reply:
I'm not sure what effects does switching to nodejs8 have, however if it's working with the prcheck, I don't see any issues there. we can use rh-nodejs8 instead.

Reply:
I'm not sure what effects does switching to nodejs8 have, however if it's working with the prcheck, I don't see any issues there. we can use rh-nodejs8 instead.

Reply:
I'm not sure what effects does switching to nodejs8 have, however if it's working with the prcheck, I don't see any issues there. we can use rh-nodejs8 instead.

Reply:
I'm not sure what effects does switching to nodejs8 have, however if it's working with the prcheck, I don't see any issues there. we can use rh-nodejs8 instead.